src/benchmarks/micro/libraries/System.Private.Xml/Perf.XmlSerializationWriter.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Text;
+using System.Xml.Serialization;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Xml.Tests
+{
+	[BenchmarkCategory(Categories.Libraries)]
+	public class Perf_XmlSerializationWriter
+	{
+		private const int Iterations = 1000;
+
+		private readonly MyXmlSerializationWriter _writer = new MyXmlSerializationWriter();
+
+		private static readonly DateTime Now = new DateTime(2022, 9, 30, 9, 4, 15, DateTimeKind.Utc);
```

Comment:
I think these shouldn't be `static readonly` in order to prevent some JIT optimization that otherwise may occur, and to have the benchmarks more realistic (i.e. non constant inputs).